controllers/azuresql_combined_test.go

```diff
@@ -338,3 +339,47 @@ func TestAzureSqlServerCombinedHappyPath(t *testing.T) {
 	})
 
 }
+
+func TestAzureSqlServer_KeyVaultSoftDelete_CreateDeleteCreateAgain(t *testing.T) {
+	t.Parallel()
+	defer PanicRecover(t)
+	ctx := context.Background()
+	require := require.New(t)
+
+	rgLocation := "westus2"
+
+	// Create a KeyVault with soft delete enabled that we can use to perform our tests
```

Comment:
That's a _very_ good question. I have a helper method that does it for the client tests but I'm not using it here. It might not be getting set at all.

Very good eye.

Reply:
Fixed this